index.html

```diff
@@ -1266,57 +1248,29 @@ <h3>Identifiers</h3>
         </pre>
 
         <p>
-The example above uses two types of identifiers. The first identifier is for
-the <a>verifiable credential</a> and uses an HTTP-based URL. The second
-identifier is for the <a>subject</a> of the <a>verifiable credential</a> (the
-thing the <a>claims</a> are about) and uses a <a>decentralized identifier</a>,
-also known as a <a>DID</a>.
+위의 예는 두 가지 유형의 식별자를 사용합니다. 첫 번째는 <a>검증가능한 크리덴셜</a>을 위한 것이며 HTTP 기반 URL을 사용한다. 두 번째는 <a>검증가능한 크리덴셜</a>의 <a>주체</a>(<a>클레임</a>이 관련된 것)를 위한 것이며 <a>DID</a>라고도하는 <a>탈중앙 식별자</a>를 사용한다.
         </p>
 
         <p class="note">
-As of this publication, <a>DIDs</a> are a new type of identifier that are not
-necessary for <a>verifiable credentials</a> to be useful. Specifically,
-<a>verifiable credentials</a> do not depend on <a>DIDs</a> and <a>DIDs</a> do
-not depend on <a>verifiable credentials</a>. However, it is expected that many
-<a>verifiable credentials</a> will use <a>DIDs</a> and that software libraries
-implementing this specification will probably need to resolve <a>DIDs</a>.
-<a>DID</a>-based URLs are used for expressing identifiers associated with
-<a>subjects</a>, <a>issuers</a>, <a>holders</a>, credential status lists,
-cryptographic keys, and other machine-readable information associated with a
-<a>verifiable credential</a>.
+본 간행물에 의하면, <a>DIDs</a>는 편리한 <a>검증가능한 크리덴셜</a>이 필요치 않은 새로운 유형의 식별자 아다. 구체적으로 <a>검증가능한 크리덴셜</a>은 <a>DIDs</a>에 의존하지 않으며, <a>DIDs</a>는 <a>검증가능한 크리덴셜</a>에 의존하지 않는다. 그렇지만 많은 <a>검증가능한 크리덴셜</a>이 <a>DIDs</a>를 사용할 것으로 예상되고, 이 규격을 구현하는 소프트웨어 라이브러리는 <a>DIDs</a>를 구현할 필요가 있을 것이다. <a>DID</a> 기반 URL은 <a>주체</a>, <a>발급자</a>, <a>보유자</a>, 크리덴셜 상태 목록, 암호화 키 그리고 <a>검증가능한 크리덴셜</a>과 관련된 기계 판독 가능 정보를 관련된 식별자로 표현하는 데 사용된다.
```

Comment:
```suggestion
본 간행물에 의하면, <a>DIDs</a>는  새로운 유형의 식별자이지만 <a>검증가능한 크리덴셜</a>이 유용해지기 위해 필수적이지는 않다. 구체적으로 <a>검증가능한 크리덴셜</a>은 <a>DIDs</a>에 의존하지 않으며, <a>DIDs</a>는 <a>검증가능한 크리덴셜</a>에 의존하지 않는다. 그렇지만 많은 <a>검증가능한 크리덴셜</a>이 <a>DIDs</a>를 사용할 것으로 예상되고, 이 규격을 구현하는 소프트웨어 라이브러리는 <a>DIDs</a>를 구현할 필요가 있을 것이다. <a>DID</a> 기반 URL은 <a>주체</a>, <a>발급자</a>, <a>보유자</a>, 크리덴셜 상태 목록, 암호화 키 그리고 <a>검증가능한 크리덴셜</a>과 관련된 기계 판독 가능 정보를 관련된 식별자로 표현하는 데 사용된다.
```

직역을 하면 어려워서 문장의 구성을 조금 바꿔봤습니다